planet/cli/orders.py

```diff
@@ -263,6 +263,12 @@ async def create(ctx, request: str, pretty):
     type=types.JSON(),
     help="""Credentials for cloud storage provider to enable cloud delivery of
     data. Can be a json string, filename, or '-' for stdin.""")
+@click.option(
+    '--stac',
+    default=False,
```

Comment:
I think a default of `True` here makes sense (with a `--no-stac` option to disable).  Having a standard metadata format should provide more forward compatibility – someone could upgrade to our future Orders v3 API and ideally still use the same STAC metadata.

Reply:
I'm +1 on making it default, especially since it's just an extra file that won't break anything. The one argument 'against' doing so would be that the STAC output from orders is still 'beta'. But so is the Python SDK, so I say we push to help get the stac output from orders to out of beta by the time we release the Python SDK as 2.0.0.

Reply:
Sounds good, changed to STAC metadata as default, and changed flag accordingly (`--stac` ➡️  `--no-stac`)